fe1-web/store/reducers/SocialReducer.ts

```diff
@@ -175,6 +175,33 @@ export const makeChirpsList = () => createSelector(
   },
 );
 
+export const makeChirpsListOfUser = (user: PublicKey | string) => {
+  const userPublicKey = user.valueOf();
+  return createSelector(
+    // First input: Get all chirps across all LAOs
+    (state) => getSocialState(state),
+    // Second input: Get the current LAO id,
+    (state) => getLaosState(state).currentId,
+    (chirpList: SocialLaoReducerState, laoId: string | undefined): ChirpState[] => {
+      if (!laoId || !userPublicKey) {
+        return [];
+      }
+      const laoChirps = chirpList.byLaoId[laoId];
+      if (laoChirps) {
+        const allUserChirps: ChirpState[] = [];
+        const userChirps = laoChirps.byUser[userPublicKey];
+        if (userChirps) {
+          userChirps.forEach(
+            (id: string) => allUserChirps.push(chirpList.byLaoId[laoId].byId[id]),
+          );
+          return allUserChirps;
+        }
+      }
+      return [];
+    },
+  );
+};
```

Comment:
I would move this to the respective reducer, for coherence with the other selectors who are all there.

Reply:
I'm not sure I understand where I should put this. Isn't it already at the right place ? `makeChirpsList()` is right above it.

Reply:
You're absolutely right, it's in the right place.

My brain had a segfault 😅 I was somehow convinced that this was in a React Component ... I don't even know how or why 🤷